dpctl/memory/_memory.pyx

```diff
@@ -751,11 +751,29 @@ def as_usm_memory(obj):
         )
 
 
-cdef api DPCTLSyclUSMRef get_usm_pointer(_Memory obj):
+cdef api DPCTLSyclUSMRef Memory_get_usm_pointer(_Memory obj):
```

Comment:
For such functions, can we have the convention we follow in `libsyclinterface`, *i.e.*, `Memory_GetUsmPointer`?